sites/diverseeducation.com/server/templates/website-section/awards-honors/top-women/index.marko

```diff
@@ -116,7 +116,7 @@ $ const queryParams = {
             flush=true
             full-height=true
             with-teaser=false
-            with-section=false
+            with-section=falseb
```

Comment:
Is this a typo/stray character addition here?

Reply:
I haven't edited that part of the code, anyway, it's been corrected.

Reply:
I updated the section-with-children.js file, still doesn't look right.

Reply:
This stray character is still here.